docs/proposals/001-initial-proposal/001-initial-proposal.md

```diff
@@ -0,0 +1,209 @@
+
+# LLM Instance Gateway
+<!-- toc -->
```

Comment:
Recommend adding some kind of status close to the top of each proposal. Maybe we should also have a template for proposals?

Some x-refs:

1. https://gateway-api.sigs.k8s.io/geps/overview/
1. https://github.com/kubernetes-sigs/gateway-api/tree/main/geps/gep-696
1. https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template

Reply:
> Recommend adding some kind of status close to the top of each proposal.
Done

> Maybe we should also have a template for proposals?
This did attempt to loosely follow the KEP pattern, the third option you list. 
Since much of this is still TBD, a good portion of the KEP contents were removed. Definitely open to a more structured standard.